recipes/irida-linker/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+
+cp ngsArchiveLinker.pl $PREFIX/bin
+
+#testing usage statement
+perl ngsArchiveLinker.pl
```

Comment:
Can this go away, given that ngsArchiveLinker.pl is already tested in meta.yaml?

Reply:
It went away! Is already tested in meta.yaml